code/modules/research/xenobiology/vatgrowing/petri_dish.dm

```diff
@@ -26,7 +26,8 @@
 	if(!sample || !istype(A, /obj/structure/sink))
 		return FALSE
 	to_chat(user, span_notice("You wash the sample out of [src]."))
-	sample = null
+	QDEL_NULL(sample)
+	update_appearance()
```

Comment:
should be handled in handle_atom_del instead.
```
/obj/item/petri_dish/handle_atom_del(atom/deleting_atom)
	if(deleting_atom == sample)
		sample = null
		update_appearance()
	return ..()
```

Reply:
done